index2.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <script defer src="app.js"></script> -->
+    <link rel="stylesheet" href="style.css">
```

Comment:
### Optional
- [ ] Why do we have multiple repeated files in our project? This makes our code very messy and difficult to understand. Kindly delete all files that are not needed for this project. For example; we have index2.html and index.html, which one are we using? Also what is the app.js purpose in the project?

Reply:
Deleted all unnecessary files